lib/tpm2_identity_util.c

```diff
@@ -218,9 +219,9 @@ bool tpm2_identity_util_encrypt_seed_with_public_key(TPM2B_DIGEST *protection_se
                 parent_pub, label, labelLen, encrypted_protection_seed);
         break;
     case TPM2_ALG_ECC:
-        LOG_ERR("Algorithm '%s' not supported yet", tpm2_alg_util_algtostr(alg,
-            tpm2_alg_util_flags_any));
-        result = false;
+        result = ecdh_derive_seed_and_encrypted_seed(parent_pub,
+                label, labelLen,
```

Comment:
@t-higuchi 
Please rebase to current master. no camelCase, use label_len instead.